docs/OnnxReleases.md

```diff
@@ -45,13 +63,52 @@ repository=https://test.pypi.org/legacy/
 username=<username>
 password=<password>
 ```
-* Do ``twine upload dist/* -r pypitest`` to upload it to the test instance of pypi.
-* After uploading to pypitest, you can test the source distribution by doing ``pip install –index-url https://test.pypi.org/simple/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
-* *NOTE - Once a distribution is uploaded to pypi, you cannot overwrite it on the same pypi instance.*
-* Once completely verified do, ``twine upload dist/*``  to upload to the official pypi.
-* *Conda - *
-    * Conda builds of ONNX are done via conda-forge, which runs infrastructure for building packages and uploading them to conda-forge. You need to submit a PR to https://github.com/conda-forge/onnx-feedstock (see https://github.com/conda-forge/onnx-feedstock/pull/1/files for an example PR.) You will need to have uploaded to PyPi already, and update the version number and tarball hash of the PyPi uploaded tarball.
-* The release branch needs to be marked as a protected branch to preserve it. The branch corresponding to each release must be preserved. (You may need to ask an admin to do this)
+* Do ``twine upload dist/* pypitest`` to upload it to the test instance of PyPI.
+* After uploading to PyPItest, you can test the source distribution by doing ``pip install --index-url https://test.pypi.org/legacy/ onnx`` in a new environment. Test this installation with different environments and versions of protobuf binaries.
+
+## PyPI package verification
+**Test ONNX itself**
+* Test the PyPI package installation with different combinations of various Python versions, Protobuf versions and platforms.
+* After installing the PyPI package, run `pytest` in the release branch and test the ONNX basic functions like `load`, `checker.check_model` and `shape_inference.infer_shapes` on certain example ONNX model.
+
+**Test with onnxruntime**
+* onnxruntime uses ONNX as submodule so it cannot be tested with ONNX PyPI package. Instead, onnxruntime can be tested with ONNX's release commit. Update the commit ID in onnxruntime and check the status of CIs from onnxruntime.
+* To test the interaction with onnxruntime, use ONNX functions like `load`, `checker.check_model`, `shape_inference.infer_shapes`, `save` with onnxruntime functions like `InferenceSession` and `InferenceSession.run` on certain example ONNX model.
+
+**Test with ONNX converters**
+* Cooperate with the converter teams. Provide them with the produced ONNX PyPI packages and let converter teams use them with their converters to check whether there is any issue.
```

Comment:
change this to 
Create github issue in converters repo to have them test the pypi test package. and add the links to converter repos.

Reply:
I don't quite understand about this line... Is this something like the following sentence?
Create GitHub issues in converters repos to provide them the package links and have them test the TestPyPI packages.

Reply:
right... and add the links to all the converter repos... you can also add an option to announce it on the converter channel on slack